templates/al2/provisioners/install-worker.sh

```diff
@@ -196,7 +196,9 @@ EOF
 ###############################################################################
 
 sudo yum install -y nerdctl
-sudo mkdir /etc/nerdctl
+if [ ! -d "/etc/nerdctl" ]; then
+    sudo mkdir /etc/nerdctl
+fi
```

Comment:
oh 🥴 this probably should've been
```suggestion
sudo mkdir -p /etc/nerdctl
```